tests/Integration/Behaviour/Features/Scenario/Product/SpecificPrice/specific_price_priorities.feature

```diff
@@ -8,88 +8,104 @@ Feature: Set Specific Price priorities from Back Office (BO).
   As an employee I want to be able to set specific price priorities to single product and to all products
 
   Scenario: I set specific price priorities to single product
-    Given I add product "product1" with following information:
+    Given default specific price priorities are set to following:
```

Comment:
We need another check that asserts that custom product priorities are not set (relying on the `GetProductForEditing` query)